File: tests/expected/math_func.zig
const std = @import("std");
const print = std.debug.print;
pub fn main_func() void {
    const a: i32 = std.math.pow(i32, 2, 4);
    print("{}\n", .{a});
}

pub fn main() void {
    main_func();
}

File: tests/expected/math_func.dart
// @dart=3.4
import 'dart:math';
import 'package:sprintf/sprintf.dart';

main_func() {
  final int a = (pow(2, 4) as int);
  print(sprintf("%s", [a]));
}

main(List<String> argv) {
  main_func();
}
